nomic/utils.py

```diff
@@ -243,30 +243,48 @@ def get_object_size_in_bytes(obj):
 # Helpful function for downloading feather files
 # Best for small feather files
 def download_feather(
-    url: Union[str, Path], path: Path, headers: Optional[dict] = None, retries=3, overwrite=False
+    url: Union[str, Path], path: Path, headers: Optional[dict] = None, num_attempts=3, overwrite=False
```

Comment:
Should we just remove auto-retry on this? It seems like a strange feature to include in a library like this. If a download fails, it fails. They can retry now.

Reply:
hmm, let's keep it for now, but I can bump num_attempts down to 1.